governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,66 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3]
+    abstain: [0]
+  victory_conditions:
+    - { 
+        type: 'and', 
+        conditions: [
+          { type : approval },
+          { type: majority, percent: 50 },
+        ]
+      }
+    - { type : default, value : $StatusQuoOrRejectOptionNumber }
+  result_display: approval-breakdown
+options:
+   0: Abstain
+   1: $Option1
+   2: $Option2
+   3: $Option3
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Poll:$GenericChange - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should select all the options that you would support in an executive vote.**
+- **Unselected options signal that you do not approve of those options and would not support the option(s) in an executive vote.**
+- **If you wish to abstain, that should be your one and only choice.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+* $GenericChangeA
+* $GenericChangeB
+* $GenericChangeC
+
+Please review the discussion [thread]($discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**The winner of this vote requires an overall majority of votes to support it. The non-`Abstain` option with the most votes AND the support of a majority of votes is deemed the winner and the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
```

Comment:
Not accurate? "If the votes for the 'No' option equal or exceed the votes for all other options, or if no other option reaches a majority then no further action will be taken at this time.**"

Alternatively:

**If the condition above is not met, then no further action will be taken at this time.**